client/app/license/order/license-order.html

```diff
@@ -237,12 +237,13 @@ <h2 data-translate="license_order_step4_title"></h2>
                         </div>
                     </div>
 
-                    <div class="col-md-3"
+                    <div class="col-md-12"
```

Comment:
From the context, seems weird to have `col-md-9` then `col-md-12` then `col-md-3`. If you want to achieve that I think you should have separated rows

Reply:
Also, `col-md-12` is basically useless,  you are just saying that your block item should behave like a block item

Reply:
Yup, have replaced it as rows. Hope this is okay.